### PR TITLE
python3Packages.websocket-client: 0.58.0 -> 1.1.0

### PR DESCRIPTION
**Pull request by drewrisinger**
###### Motivation for this change

Update websocket-client to latest version. Found update when trying to update downstream package that requires newest version.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/websocket-client/default.nix [hunk 1]
```diff
@@ -2,26 +2,23 @@
 , backports_ssl_match_hostname
```
Comment (dotlambda) [inline pkgs/development/python-modules/websocket-client/default.nix:2]:
```suggestion
```

File: pkgs/development/python-modules/websocket-client/default.nix [hunk 2]
```diff
@@ -2,26 +2,23 @@
...
 , buildPythonPackage
 , fetchPypi
-, isPy27
+, pythonOlder
 , pytestCheckHook
 , six
```
Comment (dotlambda) [inline pkgs/development/python-modules/websocket-client/default.nix:7]:
```suggestion
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/websocket-client/default.nix (hunk 1 above)
Comment (drewrisinger) [reply pkgs/development/python-modules/websocket-client/default.nix:2]: Thanks.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 127099` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
8 packages marked as broken and skipped:
</summary>
<ul>
<li>awsebcli</li>
<li>mlflow-server</li>
<li>python38Packages.flower</li>
<li>python38Packages.mlflow</li>
<li>python38Packages.rl-coach</li>
<li>python39Packages.flower</li>
<li>python39Packages.mlflow</li>
<li>python39Packages.rl-coach</li>
</ul>
</details>
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>tts: log was empty</li>
  </ul>
</details>
<details>
<summary>
121 packages built:
</summary>
<ul>
<li>appdaemon</li>
<li>arion</li>
<li>aws-sam-cli</li>
<li>azure-cli</li>
<li>buildbot (python38Packages.buildbot)</li>
<li>buildbot-full (python38Packages.buildbot-full)</li>
<li>buildbot-ui (python38Packages.buildbot-ui)</li>
<li>docker-compose</li>
<li>errbot</li>
<li>fdroidserver</li>
<li>giara</li>
<li>home-assistant</li>
<li>jellyfin-mpv-shim</li>
<li>ledger_agent (python38Packages.ledger_agent)</li>
<li>mpfshell</li>
<li>octoprint</li>
<li>onionshare</li>
<li>onionshare-gui</li>
<li>opsdroid</li>
<li>patroni</li>
<li>plex-mpv-shim</li>
<li>python38Packages.aioambient</li>
<li>python38Packages.aria2p</li>
<li>python38Packages.celery</li>
<li>python38Packages.devolo-home-control-api</li>
<li>python38Packages.django-raster</li>
<li>python38Packages.djmail</li>
<li>python38Packages.docker</li>
<li>python38Packages.docker-py</li>
<li>python38Packages.dockerspawner</li>
<li>python38Packages.flask-socketio</li>
<li>python38Packages.gradient</li>
<li>python38Packages.homematicip</li>
<li>python38Packages.ibm-watson</li>
<li>python38Packages.jellyfin-apiclient-python</li>
<li>python38Packages.jupyter-repo2docker</li>
<li>python38Packages.kubernetes</li>
<li>python38Packages.ledgerblue</li>
<li>python38Packages.libsoundtouch</li>
<li>python38Packages.moto</li>
<li>python38Packages.openshift</li>
<li>python38Packages.plexapi</li>
<li>python38Packages.praw</li>
<li>python38Packages.pushbullet</li>
<li>python38Packages.pyfireservicerota</li>
<li>python38Packages.python-engineio</li>
<li>python38Packages.python-engineio_3</li>
<li>python38Packages.python-mpv-jsonipc</li>
<li>python38Packages.python-socketio</li>
<li>python38Packages.python-socketio_4</li>
<li>python38Packages.roonapi</li>
<li>python38Packages.runway-python</li>
<li>python38Packages.samsungctl</li>
<li>python38Packages.samsungtvws</li>
<li>python38Packages.slack-sdk</li>
<li>python38Packages.slackclient</li>
<li>python38Packages.sopel</li>
<li>python38Packages.tensorboardx</li>
<li>python38Packages.tern</li>
<li>python38Packages.waterfurnace</li>
<li>python38Packages.websocket-client</li>
<li>python39Packages.aioambient</li>
<li>python39Packages.buildbot</li>
<li>python39Packages.buildbot-full</li>
<li>python39Packages.buildbot-ui</li>
<li>python39Packages.celery</li>
<li>python39Packages.devolo-home-control-api</li>
<li>python39Packages.django-raster</li>
<li>python39Packages.djmail</li>
<li>python39Packages.docker</li>
<li>python39Packages.docker-py</li>
<li>python39Packages.dockerspawner</li>
<li>python39Packages.flask-socketio</li>
<li>python39Packages.gradient</li>
<li>python39Packages.homematicip</li>
<li>python39Packages.ibm-watson</li>
<li>python39Packages.jellyfin-apiclient-python</li>
<li>python39Packages.jupyter-repo2docker</li>
<li>python39Packages.kubernetes</li>
<li>python39Packages.ledger_agent</li>
<li>python39Packages.ledgerblue</li>
<li>python39Packages.libsoundtouch</li>
<li>python39Packages.moto</li>
<li>python39Packages.openshift</li>
<li>python39Packages.plexapi</li>
<li>python39Packages.praw</li>
<li>python39Packages.pushbullet</li>
<li>python39Packages.pyfireservicerota</li>
<li>python39Packages.python-engineio</li>
<li>python39Packages.python-engineio_3</li>
<li>python39Packages.python-mpv-jsonipc</li>
<li>python39Packages.python-socketio</li>
<li>python39Packages.python-socketio_4</li>
<li>python39Packages.roonapi</li>
<li>python39Packages.runway-python</li>
<li>python39Packages.samsungctl</li>
<li>python39Packages.samsungtvws</li>
<li>python39Packages.slack-sdk</li>
<li>python39Packages.slackclient</li>
<li>python39Packages.sopel</li>
<li>python39Packages.tensorboardx</li>
<li>python39Packages.tern</li>
<li>python39Packages.waterfurnace</li>
<li>python39Packages.websocket-client</li>
<li>sourcehut.buildsrht</li>
<li>sourcehut.coresrht</li>
<li>sourcehut.dispatchsrht</li>
<li>sourcehut.gitsrht</li>
<li>sourcehut.hgsrht</li>
<li>sourcehut.hubsrht</li>
<li>sourcehut.listssrht</li>
<li>sourcehut.mansrht</li>
<li>sourcehut.metasrht</li>
<li>sourcehut.pastesrht</li>
<li>sourcehut.todosrht</li>
<li>streamlink</li>
<li>streamlink-twitch-gui-bin</li>
<li>ulauncher</li>
<li>urlwatch</li>
<li>weechatScripts.wee-slack</li>
<li>zeronet</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
@ofborg build home-assistant

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 127099` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>home-assistant: <a href="https://termbin.com/cubr">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/510f9b0c51bf7e6e4daf99031bde3861/raw/index.html">pretty log </a></li>
  </ul>
</details>
